src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -789,6 +806,16 @@ protected theorem extractLsb_ofNat (x n : Nat) (hi lo : Nat) :
     getLsbD (extractLsb hi lo x) i = (i ≤ (hi-lo) && getLsbD x (lo+i)) := by
   simp [getLsbD, Nat.lt_succ]
 
+@[simp] theorem getMsbD_extractLsb {hi lo : Nat} {x : BitVec w} {i : Nat} :
+    (extractLsb hi lo x).getMsbD i =
+    (decide (i < hi - lo + 1) && (decide (lo + (hi - lo - i) < w) && x.getMsbD (w - 1 - (lo + (hi - lo - i))))) := by
```

Comment:
```suggestion
    (extractLsb hi lo x).getMsbD i 
    = (decide (i < hi - lo + 1) 
      && (decide (lo + (hi - lo - i) < w) 
      && x.getMsbD (w - 1 - (lo + (hi - lo - i))))) := by
```
Again, see previous comment on `=` style. You might want to check the exact number of spaces, as I don't trust the GitHub edit box to handle the monospacing properly